apps/dashboard/web/polaris_web/web/src/apps/dashboard/pages/settings/integrations/Postman.jsx

```diff
@@ -17,18 +17,19 @@ function Postman() {
     }
     
     async function fetchPostmanCred() {
-      let postmanData = await settingFunctions.getPostmanCredentials();
-      let postmanCred = postmanData.postmanCred
-      if (postmanCred['api_key'] && postmanCred['workspace_id']) {
-        setPostmanKey(postmanCred.api_key);
-        setSelected(postmanCred.workspace_id);
-        fetchWorkSpaces();
-      }
+      await settingFunctions.getPostmanCredentials().then((resp)=> {
```

Comment:
using async/await along with .then is an anti-pattern. They are generally meant to be used in place of each other. Ref: https://javascript.info/async-await